CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 1.15.0 - 2023-01-XX
+
+- [#208](https://github.com/php-http/discovery/pull/208) - Add composer plugin to auto-install missing implementations
```

Comment:
```suggestion
- [#208](https://github.com/php-http/discovery/pull/208) - Add composer plugin to auto-install missing implementations if php-http/discovery is directly required in the application
```

want to defuse potential backlash against this functionality.

Reply:
that wouldn't be accurate :)
the point of the plugin is precisely to allow libraries to *not* require any specific implementation and still be working out of the box, even if the root app doesn't have a matching implementation.
The requirement is that *libraries* must opt-in the plugin (apps can opt-out by deciding to disable the plugin)

Reply:
another way to opt-out for apps is just to require the implementation they prefer of course.

Reply:
ah right.  would this be accurate?

```suggestion
- [#208](https://github.com/php-http/discovery/pull/208) - Add composer plugin to auto-install missing implementations.
  When libraries require a http implementation but no library providing that implementation is installed in the application, the plugin will automatically install an implementation.
  This is only done for libraries that directly require php-http/discovery to avoid unexpected dependency installation.
```